lnwallet/channel.go

```diff
@@ -3425,7 +3425,7 @@ func (lc *LightningChannel) SignNextCommitment() (lnwire.Sig, []lnwire.Sig, []ch
 		// jobs.
 		if jobResp.Err != nil {
 			close(cancelChan)
-			return sig, htlcSigs, nil, err
+			return sig, htlcSigs, nil, jobResp.Err
```

Comment:
How does this relate to not enough signatures being produced? Is it that the daemon is shutting down, so the request returns an error, but this is actually `nil` making it think we got all the signatures needed?

Reply:
I caught this in a unit test where the mock signer returned an error. This bubbled up as a 'not enough signatures' messages without this fix.